From 59934741d94768586d2d13cf0bad6f0d2b6a7ae6 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Marc-Andr=C3=A9=20Lureau?= <marc-andre.lureau@nokia.com>
Date: Thu, 25 Jun 2009 13:13:43 +0300
Subject: [PATCH 67/72] stream-restore: initialize userdata to 0

This avoid crash when unloading and some slots are != NULL
---
 src/modules/module-stream-restore.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/modules/module-stream-restore.c b/src/modules/module-stream-restore.c
index 8e96932..030e167 100644
--- a/src/modules/module-stream-restore.c
+++ b/src/modules/module-stream-restore.c
@@ -1107,7 +1107,7 @@ int pa__init(pa_module*m) {
     if (!restore_muted && !restore_volume && !restore_device)
         pa_log_warn("Neither restoring volume, nor restoring muted, nor restoring device enabled!");
 
-    m->userdata = u = pa_xnew(struct userdata, 1);
+    m->userdata = u = pa_xnew0(struct userdata, 1);
     u->core = m->core;
     u->module = m;
     u->save_time_event = NULL;
-- 
1.6.3.1

